<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>database</class>
 <widget class="QMainWindow" name="database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="computericon.qrc">
    <normaloff>:/Icons/Computericon/danni_bicon.png</normaloff>:/Icons/Computericon/danni_bicon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fff;
border: 0px;
font: 10pt &quot;Ubuntu&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>72</width>
    <height>72</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Search_edit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 2px solid gray;
 border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>251</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #56c2ea;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #47546f;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #47546f;
color: #fff;</string>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff;</string>
     </property>
     <property name="text">
      <string>Are you sure you wish to delete?</string>
     </property>
    </widget>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>label</zorder>
    <zorder>frame_persons</zorder>
   </widget>
   <widget class="QFrame" name="frame_computers">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #56c2ea;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonComputersInsert">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #47546f;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonCloseComputers">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #47546f;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Sorry no</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Computers_name">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;type name here&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Computers_yearbuilt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;year built&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Computers_WasBuilt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>99</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Was built</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboComputerstype">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>251</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #fff;</string>
     </property>
     <item>
      <property name="text">
       <string>Electronics</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mechanical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electro-mechanical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transistors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="connectText">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
color: #fff;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Connecting computer:&lt;br /&gt;Bleble&lt;br /&gt;&lt;br /&gt;with person:&lt;br /&gt;?&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_persons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #56c2ea;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonPersonInsert">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #47546f;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonClosePerson">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #47546f;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Sorry no</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="comboPersonMale">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>119</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="comboPersonFemale">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>119</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Persons_name">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;type name here&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Persons_yearborn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;born this year&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Persons_StillAlive">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>99</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Still alive</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Persons_whendied">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;died this year&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Insert"/>
    <addaction name="separator"/>
    <addaction name="action_Remove"/>
    <addaction name="actionConnect"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Insert">
   <property name="text">
    <string>&amp;Insert</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionYear_born">
   <property name="text">
    <string>Year born</string>
   </property>
  </action>
  <action name="actionYear_dead">
   <property name="text">
    <string>Year dead</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="computericon.qrc"/>
 </resources>
 <connections/>
</ui>
